scripts/test.sh

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env bash
-#
-# A script to run the tests.
```

Comment:
Actually, we were using this script to run all of the tests on the local, before checking them on CI. There are certain go test flags, which are easy to miss out otherwise while running tests.

Reply:
Hi @sacOO7 please be aware that if you were using `test.sh` to run all the tests on this project, this would take twice as long to run all the tests as it now does to run them in CI (where they run in parallel as each suite has it's own runner).

The commands to run the test can always be copied directly from the pipeline's GitHub action .yml files. The project should not have to rely on a bash script when the pipeline can be expressed directly in the GitHub action .yml file. Please see #530 for more information and let me know if you still have any objection to the removal of this script.

Reply:
`The commands to run the test can always be copied directly from the pipeline's GitHub action .yml files`. If we are planning to do this, can we update the `README` file for the same? I can see the `README` file section -> https://github.com/ably/ably-go#integration-tests still mentions the use of `scripts/test.sh`. Also, it will be great if we mention relevant flags as specified by the runner.

Reply:
Good spot with the readme file. I've just pushed an update to remove reference to `test.sh` and add the commands to run the tests.

Reply:
Nice work 👍